src/audio/ffmpegaudio.cc

```diff
@@ -260,7 +260,6 @@ void DecoderContext::closeCodec()
   // a crash in ffmpeg
   if ( audioStream_ && codecContext_ && codec_ ) {
     audioStream_->discard = AVDISCARD_ALL;
-    avcodec_close( codecContext_ );
     avcodec_free_context( &codecContext_ );
```

Comment:
`avcodec_close` is `avcodec_free_context` but without desotrying the `codecContext_` itself.

The `avcodec_free_context` does `avcodec_close` and more. So just delete it is fine.

https://ffmpeg.org/doxygen/7.0/group__lavc__core.html#gab7f632082a10fcb051fa3e2db7f86942